src/main/java/com/google/cloud/tools/gradle/appengine/core/CheckCloudSdkTask.java

```diff
@@ -50,6 +51,10 @@ public void checkCloudSdkAction() {
     }
 
     String version = toolsExtension.getCloudSdkVersion();
+    if (Strings.isNullOrEmpty(version)) {
+      throw new GradleException("SDK version must be specified for validation.");
```

Comment:
Again, adding this until we figure out how to properly prevent running the task manually (should never happen otherwise).